### ISSUE TITLE
Open testing error 6 – onSpawnedArrival

### ISSUE THREAD

**Issue by Bob-Murphy** [official-antistasi-community/A3-Antistasi#548]
**### Please assign yourself to the issue or comment your name when you start working on it.**

This error occurred whilst the second open testing – Kunduz – RPT on discord – lines 28191 and following

```
15:59:53 Error in expression <awn A3A_fnc_airdrop;
};
};
}
else
{

if(_cargoGroup != grpNull) then
{
_wp0 = _v>
15:59:53   Error position: <_cargoGroup != grpNull) then
{
_wp0 = _v>
15:59:53   Error Undefined variable in expression: _cargogroup
15:59:53 File functions\Convoy\fn_onSpawnedArrival.sqf [A3A_fnc_onSpawnedArrival]..., line 51
```

**Comment by wurzel0701** [comment]
I assume it has something to do with the joinGroups error, which cause should be fixed by #544

**Comment by wurzel0701** [comment]
Opened a second PR to fix the symptom of the other error, it should no longer throw errors
